src/app/pages/fund/choose-asset-to-fund/choose-asset-to-fund.tsx

```diff
@@ -44,12 +36,14 @@ export function ChooseCryptoAssetToFund() {
 
   const filteredCryptoAssetBalances = useMemo(
     () =>
-      cryptoCurrencyAssetBalances.filter(assetBalance =>
-        whenWallet({
-          ledger: checkBlockchainAvailable(assetBalance.blockchain),
-          software: true,
-        })
-      ),
+      cryptoCurrencyAssetBalances
+        .filter((assetBalance): assetBalance is CryptoAssetBalance => assetBalance != null)
```

Comment:
Can we avoid the single equals expressions here? I know this check also checks for undefined but is very unobvious. We don't use elsewhere in app. 

Could use `isDefined` instead?

Reply:
Good point, done!